src/mnode/mnode-controller.h

```diff
@@ -76,7 +76,7 @@ class CMasterNodeController
 
     int MasternodeCheckSeconds, MasternodeMinMNBSeconds, MasternodeMinMNPSeconds, MasternodeExpirationSeconds, MasternodeWatchdogMaxSeconds, MasternodeNewStartRequiredSeconds;
     int MasternodePOSEBanMaxScore;
-
+    int MasternodeWeekBySeconds;
```

Comment:
This timeout variable meaning is MN expiration time for new_start_required status. I think MasterNodeWeekBySeconds does not reflect that meaning. You can call it smth like MNStartRequiredExpirationTime.
- make comments always when you add new member variable